### PR TITLE
admins can now choose to announce filenames on sounds played globally

### LINKED ISSUE #4771: When an admin plays a .midi./ogg, the filename should be displayed to every player.
Could save on everybody saying 'what song is this'

### PR DESCRIPTION
**Pull request by Intigracy**
fixes #4771

### CODE REVIEW

File: code/modules/admin/verbs/playsound.dm [hunk 1]
```diff
@@ -7,6 +7,9 @@
 	uploaded_sound.status = SOUND_STREAM | SOUND_UPDATE
 	uploaded_sound.priority = 250
 
+	var/prompt = alert(src, "Do you want to announce the filename to everyone?","Announce?","Yes","No")
+	if(prompt == "Yes")
+		to_chat(world, "<B>[src.ckey] played sound [S]</B>")
```
Comment (PJB3005) [inline code/modules/admin/verbs/playsound.dm]: Using the `key` instead of `ckey` is probably better in this case.

Comment (Intigracy) [inline code/modules/admin/verbs/playsound.dm]: Any specific reasoning?

Comment (Intigracy) [inline code/modules/admin/verbs/playsound.dm]: Oh neat, key is properly uppercased. Thanks.

**Comment by Probe1** [comment]
clue

File: code/modules/admin/verbs/playsound.dm (hunk 1 above)
Comment (Intigracy) [inline code/modules/admin/verbs/playsound.dm]: fixed

**Pull request merged**